### PR TITLE
Fix "Open file", wrong path when using --path option

### LINKED ISSUE #452: Fix "Open file", wrong path when using --path option

### PR DESCRIPTION
**Pull request by hojelse**
Fixes #452

### CODE REVIEW

**Comment by joglr** [comment]
> Has global variable, kinda ugly, any better ideas?

Maybe try using use `(await getArgs()).path`

File: src/analyzer/analyze.server.ts [hunk 1]
```diff
@@ -205,7 +207,7 @@ export async function analyze(useCache = true) {
     setLogLevel(args.log as string)
   }
 
-  let repoDir = args.path
+  repoDir = args.path
   if (!isAbsolute(repoDir)) repoDir = resolve(process.cwd(), repoDir)
```
Comment (hojelse) [inline src/analyzer/analyze.server.ts:211]: ^ that would not use this logic

Comment (hojelse) [inline src/analyzer/analyze.server.ts:211]: ^ that would not use this logic

**Comment by hojelse** [comment]
> > Has global variable, kinda ugly, any better ideas?
> 
> Maybe try using use `(await getArgs()).path`



> > Has global variable, kinda ugly, any better ideas?
>

**Comment by hojelse** [comment]
> > Has global variable, kinda ugly, any better ideas?
> 
> Maybe try using use `(await getArgs()).path`



> > Has global variable, kinda ugly, any better ideas?
>

**Comment by hojelse** [comment]
oops

**Comment by hojelse** [comment]
oops

**Review by tjomson — APPROVED** [review]
epic 🗿🗿

File: src/analyzer/analyze.server.ts (hunk 1 above)
Comment (joglr) [inline src/analyzer/analyze.server.ts:211]: We could maybe move this check into getArgs() then?

**Pull request merged**